.github/workflows/build.yaml

```diff
@@ -28,8 +28,8 @@ jobs:
         shell: bash -l {0}
     env:
       CHANS_DEV: "-c pyviz/label/dev  -c conda-forge"
-      PKG_TEST_PYTHON: "--test-python=py37"
-      PYTHON_VERSION: "3.7"
+      PKG_TEST_PYTHON: "--test-python=py39"
+      PYTHON_VERSION: "3.9"
```

Comment:
There's a known issue on pyctdev that occurs when building a package with a base Python version greater than 3.7. You might hit that :/

Reply:
Thanks, I will revert to 3.7 for the time being then.